src/components/phase/card/PhaseCard.tsx

```diff
@@ -0,0 +1,134 @@
+import classNames from 'classnames'
+import { useCallback, useEffect, useRef, useState } from 'react'
+import { useMediaType } from '../../../hook/useMediaType'
+import type { PhaseDTO, Duration, Task } from '../../../data/phase/dto.type'
+import ClearIcon from '@mui/icons-material/Clear'
+import LoupeIcon from '@mui/icons-material/Loupe'
+import moment from 'moment'
+import Link from 'next/link'
+
+type DropDownProps = Pick<PhaseDTO, 'phaseName' | 'tasks'> & {
```

Comment:
I think you should not link the props type to the DTO's one, because if the DTO changes or evolves, you will also have to modify the props.

So, I would have declared explicitly the props type.